pkg/kubectl/describe.go

```diff
@@ -843,15 +844,6 @@ func describeContainers(label string, containers []api.Container, containerStatu
 			}
 		}
 
-		resourceToQoS := qosutil.GetQoS(&container)
-		if len(resourceToQoS) > 0 {
-			fmt.Fprintf(out, "    QoS Tier:\n")
-		}
-		for _, resource := range SortedQoSResourceNames(resourceToQoS) {
```

Comment:
[SortedQoSResourceNames()](https://github.com/kubernetes/kubernetes/blob/03fc51f74f4228825e0943a3a6e8a097063feb75/pkg/kubectl/sorted_resource_name_list.go#L65) isn't used anywhere anymore. Should we also remove it @derekwaynecarr?

Sorry for the late comment -- wasn't aware of this PR until now.